base/deprecated.jl

```diff
@@ -290,7 +290,9 @@ for f in (
 end
 # base/complex.jl
 @dep_vectorize_1arg Complex round
-@dep_vectorize_1arg Complex float
+# float(A) -> float.(A)
+@dep_vectorize_1arg Number float
+@dep_vectorize_1arg AbstractString float
```

Comment:
these aren't in the right section since these are scheduled to be deleted before 0.7 rc's, but newly modified deprecations should stay until 1.0-pre